specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -164,6 +165,201 @@ Consider at least the below topics but you do not need to cover those that are n
       * Is it logged?
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
+### Frontend components
+
+**Home Page**
+
+![](images/home.png)
+
+Contains three widgets:
+- [data
+  job](https://github.com/vmware/versatile-data-kit/wiki/dictionary#data-job)
+  execution success rate - shows the success rate for all data job
+  [executions](https://github.com/vmware/versatile-data-kit/wiki/dictionary#data-job-execution)
+  in the last 14 days
+- data jobs with failing exectuions(last 24h) - the list can also be used to
+  navigate to individual job details
+- most recent failed executions(last 24h) - the list can also be used to
+  navigate to individual failed execution details
+
+**Data Jobs Page**
+
+![](images/jobs_page.png)
+
+Shows all existing data jobs and their properties
+- Job name
+- Description
```

Comment:
nit: didn't see Description field in the > Manage >> Data Jobs Page,
and a couple of other fields...

Reply:
This is more of a problem with the screenshot. I'll re-do it in a way where it's visible.